### PR TITLE
Implement Open/Load Mp3 files

### PR DESCRIPTION
**Pull request by mazenwkamel**

### CODE REVIEW

File: .gitignore
Comment (mazenwkamel) [inline .gitignore:1]: fixed sonarlint files being pushed

**Review by ZiadElGendy — CHANGES_REQUESTED** [review]
Didn't we say we'd pass FileInfo instead of MP3 or am I misunderstanding the code

Also, on testing I found an issue with the library where you have to specify which version of id3 tag is used in the file correctly otherwise the library returns a null object (check attached code). Should this be addressed when loading or when modifying the tag?
```
private static void LoadFromDirectoryTest()
{
    var musicFiles = Mp3FileManager.LoadMp3Files("D:\\Documents\\Programming\\Group Project\\2023\\ZMBY 3\\mp3-tag-editor\\Sample MP3s");
    foreach (Mp3 musicFile in musicFiles)
    {
        Id3Tag tagNotWorking = musicFile.GetTag(Id3TagFamily.Version2X); //returns null
        Id3Tag tagWorking = musicFile.GetTag(Id3TagFamily.Version1X);
        Console.WriteLine("Title: {0}", tagWorking.Title);
        Console.WriteLine("Artist: {0}", tagWorking.Artists);
        Console.WriteLine("Album: {0}", tagWorking.Album);
    }
}
```

**Comment by mazenwkamel** [comment]
We did say that but I realized this is more closed scope than I realized. The entire class is static so these functions can be called at any given point in the program, i.e., directly before modifying.
So given a path in the MP3Manager, the manager would directly load the mp3 files then modify right away.

As for the tags, I personally think this is an issue with tags not with files so it's not the responsibility of the Mp3FileManager to check.

**Review by ZiadElGendy — APPROVED** [review]
Okay pull request approved, I'll try to make a channel for better communication on design changes

**Review by YusefZahran — APPROVED** [review]

**Pull request merged**